docs/en/tutorials/customize_rpn.md

```diff
@@ -0,0 +1,155 @@
+# Customize RPN
+
+Region proposal network(RPN) is a submodule in [Faster R-CNN](https://arxiv.org/abs/1506.01497), which generates proposals for the second stage. The two stage model series in MMDetection mostly use classic [`RPNHead`](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/mmdet/models/dense_heads/rpn_head.py) as a RPN. Actually, any detector head which generate bounding boxes can be used as a RPN. And this has been supported since MMDetection v3.0. Here we give an example to illustrate how to make FCOS as a `rpn_head` in Faster R-CNN and how to evaluate the qulity of proposals generated by this `rpn_head`.
```

Comment:
second stage of what? we should explain.

Reply:
Most two stage detectors in MMDetection use RPNHead to generate proposals as RPN.
However, any single-stage detector can serve as an RPN since their bounding box predictions can also be regarded as region proposals and thus can be refined in the R-CNN.

Reply:
Therefore, MMDetection 3.0 supports xxxx.

Here we give an example...

Reply:
Add a outline of the steps with brief description